TelegramBot.py

```diff
@@ -193,7 +193,7 @@ def _cmd_state_light(self, target_id):
         statedict = self._get_state_filtered()
         msg=""
         for entity in statedict:
-            if re.match('^light.*', entity, re.IGNORECASE):
+            if re.match('^switch.on_off_plug_in_unit|^light.*', entity, re.IGNORECASE):
```

Comment:
Is a powerplug entity always created with an entity name "switch.on_off_plug_in_unit*"? Or is this dependend from the used integration? Which integration are you using? If the entity is variable depending in the used integration and hardware we probably should add a configuration item for this instead? Something like "extend_system" for the system command. 
```suggestion
            if re.match('^switch.on_off_plug_in_unit*|^light.*', entity, re.IGNORECASE):
```

Reply:
I'm using deconz with OSRAM powerplugs (https://www.amazon.nl/gp/product/B07VCG5G7H/). They all have this name...

Reply:
ok, i still try to find an information of this kind of name is "standard" to powerplugs or depending on the used integration (e.g. Phillips HUE?). However till then this solution is fine.